### PR TITLE
Update available message enhancement

### PR DESCRIPTION
**Pull request by KozmikNano**
I have added a message that appears whenever a update is available. I haven't changed the release number in package.json  yet, so if you do commit this than change the version number like normal.

### CODE REVIEW

**Comment by MrGeorgen** [comment]
the 'boxen' package in not in package.json, that causes it to fail

**Comment by KozmikNano** [comment]
Wow I completely forgot about that let me fix that.

**Comment by KozmikNano** [comment]
Should be fixed now :)

**Comment by MrGeorgen** [comment]
ok it works now
why do need 'boxen' I do not that you use it somewhere.
Any reason why you used child_process to start 2b2w? A simple
 `require(./main.js) ` would also do it.

 I think it should also start if there is a update because sometimes a update
 could come out onnce a day and updating every time can be very annoying

**Comment by KozmikNano** [comment]
I use boxen for the box effect once there is an update. For the child process I didn’t know there was a separate command I could use. Sorry about that. I can also fix that issue you were talking about. The reason I kept it like that was because sometimes won’t be able to see the message that pops up because of they main.js displays text.

**Comment by KozmikNano** [comment]
I left a separate command for them to use a bypass within the code so they could skip it all together

**Comment by KozmikNano** [comment]
@MrGeorgen I have removed the child process code then made a feature where instead of stopping completely you can hit enter to continue.

File: check.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    let feed = await parser.parseURL('https://github.com/themoonisacheese/2bored2wait/releases.atom');
+
+
+
+    feed.items.every(item => {
+        console.log(item.title);
+        var lv = (item.title);
+        if (cv != lv) {
+	    console.log(boxen('New Update Available!', {padding: 1}));
```
Comment (KozmikNano) [inline check.js:20]: This is the line where i use the boxen dependency.

Comment (KozmikNano) [reply check.js:20]: I still haven’t changed the package.Json release number yet because there have been newer releases have came out.

**Comment by MrGeorgen** [comment]
> @MrGeorgen I have removed the child process code then made a feature where instead of stopping completely you can hit enter to continue.

Hitting enter is a smart solution, thanks.
I'll just remove a few blank lines and and fix the indentation

**Comment by Phelms215** [comment]
In docker I get stuck at this press enter to continue screen now..how would I get past this?

`
> 2bored2wait@0.1.23 start
> node check.js


                    ╭─────────────────────────────────────╮
                    │                                     │
                    │   New Update Available! → v0.1.24   │
                    │                                     │
                    ╰─────────────────────────────────────╯

Press enter to continue.
`

**Comment by KozmikNano** [comment]
I am not familiar with docker are you not able to press enter on that platform?

**Comment by themoonisacheese** [comment]
Docker is an application to containerize applications, IE you do not have to install dependencies or to mess with programs like pm2, it does it all for you. A downside is that pretty much all docker programs run in headless mode, where getting console input into them is non trivial

**Comment by KozmikNano** [comment]
Is there a way to edit files on that platform? If so I can update the readme

**Comment by KozmikNano** [comment]
Speaking about the readme I was wondering if you wanted to make the readme more user friendly? I can format it to look like this: https://github.com/KozmikNano/Github-release-check

**Pull request merged**